quodlibet/quodlibet/browsers/playlists/main.py

```diff
@@ -418,12 +418,20 @@ def __drag_data_received(self, view, ctx, x, y, sel, tid, etime, library):
 
     def _drag_data_get(self, view, ctx, sel, tid, etime):
         model, iters = self.__view.get_selection().get_selected_rows()
+
+        source_playlist = self.__get_name_of_current_selected_playlist()
+        target_playlist = _("target_playlist")
+        #print('adding', source_playlist, '-->', target_playlist)
+
         songs = []
         for itr in iters:
             if itr:
                 songs += model[itr][0].songs
+
         if tid == 0:
-            qltk.selection_set_songs(sel, songs)
+            prompt = ConfirmPlaylistDND(self, source_playlist, target_playlist)
```

Comment:
This implementation means that `PlaylistsBrowser` has a hard-coded dependency on the `ConfirmPlaylistDND` class - these are generally bad, hence big OOP projects typically use [Dependency Injection](https://en.wikipedia.org/wiki/Dependency_injection).